packages/rum-core/src/domain/foregroundContexts.spec.ts

```diff
@@ -0,0 +1,240 @@
+import { RelativeTime, relativeNow, toServerDuration, Duration } from '@datadog/browser-core'
+import { setup, TestSetupBuilder } from '../../test/specHelper'
+import { createNewEvent } from '../../../core/test/specHelper'
+import { InForegroundPeriod } from '../rawRumEvent.types'
+import { startForegroundContexts, ForegroundContexts } from './foregroundContexts'
+
+const FOCUS_PERIOD_LENGTH = 10 as Duration
+const BLUR_PERIOD_LENGTH = 5 as Duration
+
+describe('foreground context', () => {
```

Comment:
From [codecov report](https://app.codecov.io/gh/DataDog/browser-sdk/compare/854/tree/packages/rum-core/src/domain/foregroundContexts.ts), some code paths are not covered.
Should they?

Reply:
not testing feature flag false is fine by me